### PR TITLE
Fix access URLs for Disco API

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Fix access URLs for Disco API. Current ones are incorrect on this page https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-querying#access-the-discovery-api

Per [slack convo](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1713961340705989)

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Need review from PM/Eng

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 1]
```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
 There are several illustrative example queries on this page. For more examples, refer to [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples).
 
 
+## Discovery API endpoints
+
+The following are the endpoints for accessing the Discovery API. Use the one that's appropriate for your region and plan.
+
+| Deployment type |	Discovery API URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://metadata.cloud.getdbt.com/api/graphql |
```
Comment (eddowh) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md]:
```suggestion
| North America multi-tenant	|	https://metadata.cloud.getdbt.com/graphql |
```

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 2]
```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
...
+| EMEA multi-tenant	|	https://metadata.emea.dbt.com/api/graphql |
```
Comment (eddowh) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md]:
```suggestion
| EMEA multi-tenant	|	https://metadata.emea.dbt.com/graphql |
```

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 3]
```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
...
+| APAC multi-tenant	|	https://metatdata.au.dbt.com/api/graphql |
```
Comment (eddowh) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md]:
```suggestion
| APAC multi-tenant	|	https://metadata.au.dbt.com/graphql |
```

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 4]
```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
...
+| Multi-cell	| `https://YOUR_ACCOUNT_PREFIX.metadata.REGION.dbt.com/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account identifier and `REGION` with your location, which could be `us1.dbt.com`. |<br />
+| Single-tenant | `https://YOUR_ACCESS_URL.metadata/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```
Comment (eddowh) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md]:
```suggestion
| Single-tenant | `https://metadata.YOUR_ACCESS_URL/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 5]
```diff
@@ -23,24 +23,22 @@ Once you've created a token, you can use it in the Authorization header of reque
...
 
 
 ## Run queries using HTTP requests
 
-You can run queries by sending a `POST` request to the `https://metadata.YOUR_ACCESS_URL/graphql` endpoint, making sure to replace:
-* `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
+You can run queries by sending a `POST` request to the Discovery API, making sure to replace:
+* `YOUR_API_URL` with the appropriate [Discovery API endpoint](#discovery-api-endpoints) for your region and plan.
 * `YOUR_TOKEN` in the Authorization header with your actual API token. Be sure to include the Token prefix.
 * `QUERY_BODY` with a GraphQL query, for example `{ "query": "<query text>" }`
```
Comment (eddowh) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md]:
```suggestion
* `QUERY_BODY` with a GraphQL query, for example `{ "query": "<query text>", "variables": "<variables in json>" }`
```

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md [hunk 6]
```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
...
+
+The following are the endpoints for accessing the Discovery API. Use the one that's appropriate for your region and plan.
+
+| Deployment type |	Discovery API URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://metadata.cloud.getdbt.com/graphql |
+| EMEA multi-tenant	|	https://metadata.emea.dbt.com/graphql |
+| APAC multi-tenant	|	https://metadata.au.dbt.com/graphql |
+| Multi-cell	| `https://YOUR_ACCOUNT_PREFIX.metadata.REGION.dbt.com/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account identifier and `REGION` with your location, which could be `us1.dbt.com`. |<br />
+| Single-tenant | `https://metadata.YOUR_ACCESS_URL/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/discovery-querying.md:75]: does this need the `/api` in the ST URL?
```suggestion
| Single-tenant | `https://metadata.YOUR_ACCESS_URL/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```

**Review by runleonarun — APPROVED** [review]
Just one minor question but otherwise looks good!

File: website/docs/docs/dbt-cloud-apis/discovery-querying.md (hunk 6 above)
Comment (nghi-ly) [reply website/docs/docs/dbt-cloud-apis/discovery-querying.md:75]: @eddowh : wdyt?

Comment (eddowh) [reply website/docs/docs/dbt-cloud-apis/discovery-querying.md:75]: nope it doesn't. see Ally for example: https://metadata.cloud.ally.getdbt.com/graphql

**Pull request merged**